common-npm-packages/azure-arm-rest-v2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-azure-arm-rest-v2",
-  "version": "3.213.0",
+  "version": "4.215.0",
```

Comment:
1. Is it a breaking change? (is it possible to run into problems for the tasks which are using this library?)
2. What if a dependent Task is using "@types/node" with 10 but this has been upgraded 16? Is there any problem for version difference? (did you test it?)

We've just recently bumped major version to 3 due to MSAL upgrade and updating tasks accordingly with ^3.